src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -464,7 +464,7 @@ You'll be able to mock executions by configuring the path, method, headers, and
 {% /only_light %}
 
 
-## Events {% #events %}
+# Events {% #events %}
 
 Changes in Appwrite emit events. You can configure Functions to be executed in response to these events.
```

Comment:
Same thing here for the events section, we should explain that the event trigger can be found in `x-appwrite-event` header and the body will contain the data of the resource that triggered it

Reply:
Done